### DISCUSSION TITLE
Shaders in vulcan mod in the future?

### DISCUSSION THREAD

**Discussion by Pirnox** [xCollateral/VulkanMod#176] [General]
Will be in some near future way to launch shaders inside that mod like in iris shaders or optifine ?

**Comment by Splarkszter** [comment]
I would rather ask the "*insert mod name here*" maintainers to add support for the Vulkan API. Hence everyone uses OpenGL API because minecraft works inside OpenGL.

**Comment by zalvoreith-ship-it** [comment]
the only bad thing of this vulkan mod is that it doesnt have shader supporting mods

**Comment by Leastopen** [comment]
Well, now its released https://modrinth.com/mod/beryl is also by xCollateral
